src/rust/aggregator/src/data/user_history.rs

```diff
@@ -46,7 +46,10 @@ impl Data for UserHistory {
     }
 
     async fn process_and_save_historical_data(&mut self) -> DataAggregationResult {
-        self.process_and_save_internal().await
+        tracing::info!("[Aggregator] Start processing UserHistory.");
+        let res = self.process_and_save_internal().await;
+        tracing::info!("[Aggregator] Finish processing UserHistory.");
+        return res;
```

Comment:
no need to declare `res` only to immediately return it, how about just leaving this function as is?

Reply:
reverted this file